access-ocp-api-through-cis.md

```diff
@@ -0,0 +1,104 @@
+## Accessing the api endpoint deployed on OpenShift through CIS
+
+1. Setup an OpenShift cluster with ingress. The cluster comes with a default endpoint `/healthz` that is accessible as:
```

Comment:
Where do you set this up?

Reply:
We should give this link as an example - https://cloud.ibm.com/docs/openshift?topic=openshift-ingress-qs-roks4 and mention that this doc reuses an existing endpoint /healthz which comes with the default deployment

Reply:
Made the changes.